pkg/client/client.go

```diff
@@ -145,23 +147,25 @@ func (c *client) StartReportingUsage(ctx context.Context, interval time.Duration
 	return c.errorChan
 }
 
-func (c *client) sendUsage(ctx context.Context) {
+type errorHandler func(ctx context.Context, err error)
```

Comment:
What are the advantages of creating a type for `func(ctx context.Context, err error)` versus just declaring the function signature inline on L152?